recipes/json-c/all/test_v1_package/conanfile.py

```diff
@@ -1,8 +1,9 @@
-from conans import ConanFile, CMake, tools
+from conans import ConanFile, CMake
+from conan.tools.build import cross_building
```

Comment:
I think it was fine. Pure conan v1 usage, without any conan v2 stuff.

Reply:
someone should update template, it's confusing that following it is considered a bad change